packages/components/src/dropdown-menu/index.js

```diff
@@ -49,6 +50,7 @@ function DropdownMenu( {
 	menuLabel,
 	position,
 	noIcons,
+	isToolbarButton = false,
```

Comment:
Was this added for convenience? This can already be achieved with this code:
```jsx
<ToolbarItem>
  { ( toolbarItemProps ) => (
    <DropdownMenu toggleProps={ toolbarItemProps } />
  ) }
</ToolbarItem>
```

Reply:
TIL I guess I'll refactor these to avoid the prop :)